tools/serverpod_cli/lib/src/generator/dart/library_generators/entities_library_generator.dart

```diff
@@ -345,15 +344,15 @@ class SerializableEntityLibraryGenerator {
     }
   }
 
-  Method _buildEntityClassTableNameGetter(String tableName) {
+  Method _buildEntityClassTableGetter() {
     return Method(
       (m) => m
-        ..name = 'tableName'
+        ..name = 'table'
```

Comment:
Question: Isn't this message external meaning that someone could have logic based on this in their server? If so, we would need to preserve it.

Reply:
You are correct, sorry! Missed it! 👍